openjdk/excludes/ProblemList_openjdk11-openj9.txt

```diff
@@ -219,6 +219,8 @@ java/io/FileOutputStream/UnreferencedFOSClosesFd.java https://github.com/eclipse
 java/nio/Buffer/DirectBufferAllocTest.java https://github.com/eclipse-openj9/openj9/issues/4473 generic-all
 java/nio/Buffer/LimitDirectMemory.java	https://github.com/eclipse-openj9/openj9/issues/8063	generic-all
 java/nio/Buffer/LimitDirectMemoryNegativeTest.java	https://github.com/eclipse-openj9/openj9/issues/8065	generic-all
+#java/nio/channels/AsyncCloseAndInterrupt.java is excluded on z/OS due to OS limitation https://github.com/adoptium/aqa-tests/issues/6092 z/OS-s390x
+java/nio/channels/AsyncCloseAndInterrupt.java https://github.com/adoptium/aqa-tests/issues/6092 z/OS-s390x
```

Comment:
Is this a temporary exclude or a permanent exclude for JDK11?

It is a permanent exclude for JDK17. https://github.com/adoptium/aqa-tests/issues/1297#issuecomment-1579952788
https://github.com/adoptium/aqa-tests/blob/9095396570510d9f597366fa6a726b2f80a1334b/openjdk/excludes/ProblemList_openjdk17-openj9.txt#L246C48-L246C97

Reply:
This is temporary exclude, we'll be taking this issue with z/OS USS team for the fix. We'll re-enable this test in both JDK11 and JDK17 once the limitation is fixed. Thank you.

Reply:
Please remove line 222